src/client/render.tsx

```diff
@@ -50,9 +50,10 @@ export class CellOutput extends React.Component<ICellOutputProps> {
         const imgStyle: Record<string, string | number> = {};
         const divStyle: Record<string, string | number> = { overflow: 'scroll' }; // This is the default style used by Jupyter lab.
         const imgSrc = `data:${mimeType};base64,${mimeBundle[mimeType]}`;
+        const customMetadata = metadata.metadata as JSONObject | undefined;
```

Comment:
Ahh, looks like it was from the metadata.metadata change. Thanks for the fix!

Reply:
I remembered you mentioning this at triage, so it was the first thing I tried 😊 thanks for the pointer!

Reply:
I mean...I caused the bug, you shouldn't be thanking me 😆 . Just didn't realize it when I made the change. I checked widgets, but didn't notice the background regression.